test/wt_hang_analyzer/wt_hang_analyzer.py

```diff
@@ -0,0 +1,616 @@
+#!/usr/bin/env python
+"""Hang Analyzer module.
+
+A prototype hang analyzer for Evergreen integration to help investigate test timeouts
+
+1. Script supports taking dumps, and/or dumping a summary of useful information about a process
+2. Script will iterate through a list of interesting processes,
+    and run the tools from step 1. The list of processes can be provided as an option.
+
+Currenty only supports Linux. There are two issues with the MacOS and windows implementation:
+1. lldb cannot attach to processes in MacOS.
+2. Windows cannot find the debug symbols.
+"""
+
+import csv
```

Comment:
More of a style thing but Python imports can be comma-separated like this:
`import package1, package2, package3`

I don't mind if you prefer this way but seems to be a waste of vertical space.

Reply:
Fixed